packages/sw-runtime-caching/src/lib/network-first.js

```diff
@@ -21,6 +23,13 @@ import assert from '../../../../lib/assert';
  * An implementation of a [network first](https://developers.google.com/web/fundamentals/instant-and-offline/offline-cookbook/#network-falling-back-to-cache)
  * request strategy.
  *
+ * By default, `NetworkFirst` will cache [opaque responses](http://stackoverflow.com/q/39109789)
```

Comment:
Nit: Could you inverse this description. "Network first will cache 200 responses as well as opaque responses. Override with ..."